### ISSUE TITLE
Segmentation fault caused by <expansion>

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#1125]
Some recent bug (perhaps since July 9th) is causing verovio to crash with a `segmentation fault: 11` when expanding plists in `<expansion>` elements.   Somehow it seems to involve the plist interface, but I don't see any problems and don't see any recent changes in the suspicious places, such as the plist parameters.

Here is an example file that crashes the current `develop` branch of verovio:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2019-07-22T21:55:22" version="2.2.0-dev-e4897b4">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mdiv-0000001282339748">
    <score xml:id="score-0000000138263344">
     <scoreDef xml:id="scoredef-0000000516535446" midi.bpm="400">
      <staffGrp xml:id="staffgrp-0000000871497941">
       <staffDef xml:id="staffdef-0000001268910966" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5">
        <label xml:id="label-0000001885119097" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <expansion xml:id="expansion-L2F1" plist="#A #B" />
      <section xml:id="A">
       <measure xml:id="measure-L1" n="0">
        <staff xml:id="staff-0000001936246350" n="1">
         <layer xml:id="layer-L1F1N1" n="1">
          <note xml:id="note-L6F1" dur="1" oct="4" pname="c" accid.ges="n" />
         </layer>
        </staff>
       </measure>
      </section>
      <section xml:id="B">
       <measure xml:id="measure-L7" right="end">
        <staff xml:id="staff-L7F1N1" n="1">
         <layer xml:id="layer-L7F1N1" n="1">
          <note xml:id="note-L9F1" dur="1" oct="4" pname="d" accid.ges="n" />
         </layer>
        </staff>
       </measure>
      </section>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

The problem line is:

```xml
      <expansion xml:id="expansion-L2F1" plist="#A #B" />
```

If this line is removed, then there is no segmentation fault.    Removing only the `@plist` will still result in a segmentation fault, so the entire `<expansion>` has to be removed.

**Comment by wergo** [comment]
I also have the problem that lately Verovio develop crashes upon the `<expansion>` element. 

This commit from 11 July (https://github.com/rism-ch/verovio/commit/c85c897f0a63ae1e1e34ed31199cd8aa04e5e7c3) adds a <PListInterface()> at the expansion constructor, line 22–24: https://github.com/rism-ch/verovio/blob/c85c897f0a63ae1e1e34ed31199cd8aa04e5e7c3/src/expansion.cpp#L22-L24

Seems that it crashes there, but I cannot tell why.

**Comment by wergo** [comment]
Thanks, Laurent, I just see that you fixed that 2 days ago with d7f5589b4614c8cc47cb4e000c376b9d2a000953 -- many thanks!
